### PR TITLE
Added Migration UG for PullToRefresh control and content change for Expander and Accordion.

### PR DESCRIPTION
**Pull request by DiwakarVenkatesan**
Added Migration UG for PullToRefresh control and content change for Expander and Accordion.

### CODE REVIEW

File: MAUI/Pull-to-Refresh/migration.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Migrate from Xamarin SfPullToRefresh to .NET MAUI PullToRefresh | Syncfusion 
+description: Learn here all about Migrating from Syncfusion Xamarin PullToRefresh to Syncfusion .NET MAUI PullToRefresh control and more.
+platform: MAUI
+control: SfPullToRefresh
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfPullToRefresh to .NET MAUI PullToRefresh
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/migration.md:10]: .....    .NET MAUI SfPullToRefresh

File: MAUI/Pull-to-Refresh/migration.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+To migrate easier from Xamarin.Forms SfPullToRefresh to .NET MAUI PullToRefresh, we kept most of the APIs from Xamarin SfPullToRefresh in MAUI PullToRefresh. However, to maintain the consistency of API naming in .NET MAUI PullToRefresh, we renamed some of the APIs. The APIs that have been changed in MAUI PullToRefresh from Xamarin SfPullToRefresh are detailed as follows.
+
+## Namespaces 
+
+<table>
+<tr>
+<th>Xamarin SfPullToRefresh </th>
+<th>.NET MAUI PullToRefresh</th></tr>
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/migration.md:19]: .NET MAUI SfPullToRefresh

**Review by PiruthivirajM — CHANGES_REQUESTED** [review]
Please address the comments.

File: MAUI/Pull-to-Refresh/migration.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+title: Migrate from Xamarin SfPullToRefresh to .NET MAUI PullToRefresh | Syncfusion 
+description: Learn here all about Migrating from Syncfusion Xamarin PullToRefresh to Syncfusion .NET MAUI PullToRefresh control and more.
+platform: MAUI
+control: SfPullToRefresh
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfPullToRefresh to .NET MAUI PullToRefresh
+
+To migrate easier from Xamarin.Forms SfPullToRefresh to .NET MAUI PullToRefresh, we kept most of the APIs from Xamarin SfPullToRefresh in MAUI PullToRefresh. However, to maintain the consistency of API naming in .NET MAUI PullToRefresh, we renamed some of the APIs. The APIs that have been changed in MAUI PullToRefresh from Xamarin SfPullToRefresh are detailed as follows.
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/migration.md:12]: do not mention "MAUI" alone.

say ".NET MAUI SfPullToRefresh" 

image

**Review by PiruthivirajM — APPROVED** [review]

File: MAUI/Pull-to-Refresh/migration.md (hunk 1 above)
Comment (DiwakarVenkatesan) [reply MAUI/Pull-to-Refresh/migration.md]: changed

File: MAUI/Pull-to-Refresh/migration.md (hunk 2 above)
Comment (DiwakarVenkatesan) [reply MAUI/Pull-to-Refresh/migration.md]: changed

**Pull request merged**